### PR TITLE
Transformed Event "problem_graded" add code and JSON

### PR DESCRIPTION
**Pull request by imhassantariq**
**Trello Link:** [HERE](https://trello.com/c/qobqOPRV/29-problem-interaction-event-problemgraded)

**Description:** Transformed Event "problem_graded" add code and JSON

**Checks before merge:**

- [x] JSON Reviewed
- [x] Code Reviewed
- [x] Unit test passed
- [x] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/current/problem_graded.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "username": "honor",
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/current/problem_graded.json]: Please sort this alphabetically.

**Comment by imhassantariq** [comment]
@pep8speaks Keep Quiet

**Comment by imhassantariq** [comment]
@pep8speaks  suggest the diff

File: openedx/features/caliper_tracking/tests/current/problem_graded.json (hunk 1 above)
Comment (mumarkhan999) [reply openedx/features/caliper_tracking/tests/current/problem_graded.json]: done.

**Pull request merged**